README.md

```diff
@@ -316,6 +316,30 @@ When set to false, when clicking on a rail, the click event will be allowed to p
 When set to true, you can scroll the container by selecting text and move the cursor.  
 **Default: false**
 
+### theme
```

Comment:
Why? This is a parameter. All the parameters are cammelCase with a lower case at the beginning.